### PR TITLE
docs(apigateway): add documentation for stage default method throttling

### PR DESCRIPTION
**Pull request by blimmer**
It took me a good deal of time to understand how to configure default method throttling for my deployment stage. Because of the special `/*/*` path, I thought that an explicit example in the documentation could help others secure their apigateway endpoints with default method throttles, especially since the default throttles are quite high (5,000 rate limit / 10,000 burst limit).

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/@aws-cdk/aws-apigateway/README.md [hunk 1]
```diff
@@ -546,6 +546,23 @@ const api = new apigateway.RestApi(this, 'books', {
 })
 ```
 
+You can also use the `deployOptions` property to configure
```
Comment (nija-at) [inline packages/@aws-cdk/aws-apigateway/README.md]: Wondering if this would be better placed in the section 'Default Integration and Method Options', given that is a 'method option'. What do you think?

File: packages/@aws-cdk/aws-apigateway/README.md [hunk 2]
```diff
@@ -546,6 +546,23 @@ const api = new apigateway.RestApi(this, 'books', {
...
+[default method throttling](https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-request-throttling.html#apigateway-api-level-throttling-in-usage-plan)
+for the deployment stage.
+
```
Comment (nija-at) [inline packages/@aws-cdk/aws-apigateway/README.md]: I would rephrase this like below - 

You can use the `methodOptions` property on the `Stage` construct to configure [default method throttling](https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-request-throttling.html#apigateway-api-level-throttling-in-usage-plan) for the deployment stage. The following snippet configures a stage whose default throttling limits are set to 100 TPS -

<insert code snippet configuring this on a `Stage`>

Configuring this on the `deployOptions` of `RestApi` will set this on the default stage that is created, like so -

<insert your current code snippet>

Comment (blimmer) [reply packages/@aws-cdk/aws-apigateway/README.md]: Thanks for this feedback. I've made this change!

File: packages/@aws-cdk/aws-apigateway/README.md (hunk 1 above)
Comment (blimmer) [reply packages/@aws-cdk/aws-apigateway/README.md]: I considered that at first. The reason I have it here is because both of these examples are configuring options at the "stage" level, not the method level.

If you were to configure method throttling along with the examples in 'Default Integration and Method Options', that value would apply to that method in all stages. For me, personally, that was easy to figure out. Discovering how to set the defaults for the stage/deployment was what was difficult.
